transifex/push.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""
+This script can be used to automatically extract translation strings,
+commit, push, and merge them to their respective repos, and then push them to transifex.
+
+To use, export an environment variable `GITHUB_ACCESS_TOKEN`. The token requires
+GitHub's "repo" scope.
+
+Run the script from the root of this repo.
+
+    python3 transifex/push.py git@github.com:edx/course-discovery.git
+
+If you want to use a custom merge method pass the --merge-method option.
+
+    python3 transifex/push.py git@github.com:edx/course-discovery.git --merge-method rebase
+"""
+from argparse import ArgumentParser
+from utils.common import MERGE_METHODS, logger, repo_context
+
+
+# The name of the branch to use.
+BRANCH_NAME = 'update-translation-strings'
+
+# The commit message to use.
+MESSAGE = 'Update translation strings'
+
+
+def push(clone_url, merge_method=None):
+    """Extracts translations for the given repo, commits them, pushes them to GitHub, opens a PR, waits for status
```

Comment:
I'm unclear on why it's necessary for this script to do anything other than push translation files to Transifex. Could you explain to me again why merges of commits to edx-mktg with unextracted translations being allowed?